README.md

```diff
@@ -4,77 +4,66 @@
-A 'spatiotemporal asset' is any file that represents information about the earth captured in a certain space and 
-time. The initial focus is primarily remotely-sensed imagery (from satellites, but also planes, drones, balloons, etc), but 
-the core is designed to be extensible to SAR, full motion video, point clouds, hyperspectral, LiDAR and derived data like
-NDVI, Digital Elevation Models, mosaics, etc. 
-
-The goal is for all major providers of imagery and other earth observation data to expose their data as SpatioTemporal Asset 
-Catalogs, so that new code doesn't need to be written whenever a new JSON-based REST API comes out that makes its data 
-available in a slightly different way. This will enable standard library components in many languages. STAC can also be
-implemented in a completely 'static' manner, enabling data publishers to expose their data by simply publishing linked JSON
-files online.
+The SpatioTemporal Asset Catalog (STAC) family of specifications aims to standardize the way
+geospatial asset metadata is structured and queried. A "spatiotemporal asset" is any file 
+that represents information about the Earth at a certain place and time. The original focus was on scenes of satellite
+imagery, but now covers a broad variety of uses, including sources such as aircraft and drone, and data such as hyperspectral optical, synthetic aperture radar (SAR), video, point clouds, lidar, digital elevation models (DEM), vector, machine learning labels, and composites like NDVI and mosaics. STAC is intentionally designed with a minimal core and flexible extension semantics to support a broad set of use cases.
+
+STAC defines a common format (object types) and interface (web service API) to be used by providers and consumers of geospatial data.  This is advantageous to providers, as they can simply use a well-designed, standard format without needing to design their own proprietary one. This is advantageous to consumer because they can use existing libraries and tools to access metadata, instead of needing to write new code to interact with each data provider's proprietary formats and APIs. 
+
+Typically, several STAC specifications are composed together to create a catalog. The **Item**, **Catalog**, and **Collection** specifications define a minimal core of the most frequently used JSON object types. Because of the hierarchical structure between these objects, a STAC catalog can be implemented in a completely 'static' manner as a group of hyperlinked Catalog, Collection, and Item URLs, enabling data publishers to expose their data as a browsable set of files. If more complex query abilities are desired, such as spatial or temporal predicates, the [STAC API specification](https://github.com/radiantearth/stac-api-spec/) can be implemented as a web service interface to query over a group of STAC objects, usually held in a database.
```

Comment:
Maybe link in to the Item, Catalog and Collection specs at their first mention? 

This is a good intro to the STAC API spec, so may be fine to just remove the 'web service api' bit above.